docs/latex/.gitignore

```diff
@@ -0,0 +1,61 @@
+# ========== LaTeX ===============
+*.acn
+*.acr
+*.alg
+*.fls
+*.synctex(busy)
+*.fdb_latexmk
+*.aux
+*.bbl
+*.blg
+*.dvi
+*.glg
+*.glo
+*.gls
+*.idx
+*.ilg
+*.ind
+*.ist
+*.lof
+*.log
+*.lol
+*.lot
+*.maf
+*.mtc
+*.mtc1
+*.out
+*.synctex.gz
+*.toc
+*.bcf
+*.run.xml
+
+# ========== OSX ===============
+.DS_Store
+.AppleDouble
+.LSOverride
+._*
+.DocumentRevisions-V100
+.fseventsd
+.Spotlight-V100
+.TemporaryItems
+.Trashes
+.VolumeIcon.icns
+.AppleDB
+.AppleDesktop
+Network Trash Folder
+Temporary Items
+.apdisk
+
+# ========== Windows ===============
+Thumbs.db
+ehthumbs.db
+Desktop.ini
+$RECYCLE.BIN/
+*.cab
+*.msi
+*.msm
+*.msp
+*.lnk
+
+# ============ Other =============
+Imgrund_Trump_Programmentwurf.pdf
```

Comment:
Wollen wir das Dokument nicht im repo haben, falls man es kompiliert sehen möchte ohne zu clonen?

Reply:
Finde es relativ nervig, es im repo zu haben, besonders weil es in VSCode dauernd automatisch neo compiled und wenn man dann den branch wechseln will, meckert git.